app/src/main/java/org/mifos/mobilebanking/utils/MFDatePicker.java

```diff
@@ -28,7 +28,12 @@
     static Calendar calendar;
     private String startDateString;
     private long startDate;
-    private boolean isStartOrEnd;
+    private int datePickerType = ALL_DAYS;
+
+    /* Constants used to select which type of date picker is being called */
+    public static final int ENDS_CURRENT_DAY = 1; // only current day and future can be picked
```

Comment:
Rename these static fields, so that they are automatically understandable. (PREVIOUS_DAYS, FUTURE_DAYS)